// @ledgerhq/types-cryptoassets/lib/slip44.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.CoinType = void 0;
// As defined here: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
var CoinType;
(function (CoinType) {
    CoinType[CoinType["AE"] = 457] = "AE";
    CoinType[CoinType["AION"] = 425] = "AION";
    CoinType[CoinType["AKA"] = 200625] = "AKA";
    CoinType[CoinType["ALGO"] = 283] = "ALGO";
    CoinType[CoinType["APTOS"] = 637] = "APTOS";
    CoinType[CoinType["ATOM"] = 118] = "ATOM";
    CoinType[CoinType["ARK"] = 111] = "ARK";
    CoinType[CoinType["ATH"] = 1620] = "ATH";
    CoinType[CoinType["BANANO"] = 198] = "BANANO";
    CoinType[CoinType["BTC"] = 0] = "BTC";
    CoinType[CoinType["BTC_CASH"] = 145] = "BTC_CASH";
    CoinType[CoinType["BTC_GOLD"] = 156] = "BTC_GOLD";
    CoinType[CoinType["BTC_PRIVATE"] = 183] = "BTC_PRIVATE";
    CoinType[CoinType["BTC_TESTNET"] = 1] = "BTC_TESTNET";
    CoinType[CoinType["CALLISTO"] = 820] = "CALLISTO";
    CoinType[CoinType["CANTON_NETWORK"] = 6767] = "CANTON_NETWORK";
    CoinType[CoinType["CARDANO"] = 1815] = "CARDANO";
    CoinType[CoinType["CELO"] = 52752] = "CELO";
    CoinType[CoinType["CLUB"] = 79] = "CLUB";
    CoinType[CoinType["CRYPTO_ORG"] = 394] = "CRYPTO_ORG";
    CoinType[CoinType["CSPR"] = 506] = "CSPR";
    CoinType[CoinType["DASH"] = 5] = "DASH";
    CoinType[CoinType["DECRED"] = 42] = "DECRED";
    CoinType[CoinType["DEXON"] = 237] = "DEXON";
    CoinType[CoinType["DIGIBYTE"] = 20] = "DIGIBYTE";
    CoinType[CoinType["DOGE"] = 3] = "DOGE";
    CoinType[CoinType["ELASTOS"] = 2305] = "ELASTOS";
    CoinType[CoinType["ELLAISM"] = 163] = "ELLAISM";
    CoinType[CoinType["MULTIVERSX"] = 508] = "MULTIVERSX";
    CoinType[CoinType["ENERGY_WEB_CHAIN"] = 246] = "ENERGY_WEB_CHAIN";
    CoinType[CoinType["EOS"] = 194] = "EOS";
    CoinType[CoinType["ETH"] = 60] = "ETH";
    CoinType[CoinType["ETH_CLASSIC"] = 61] = "ETH_CLASSIC";
    CoinType[CoinType["EXPANSE"] = 40] = "EXPANSE";
    CoinType[CoinType["FACTOM"] = 131] = "FACTOM";
    CoinType[CoinType["FIC"] = 5248] = "FIC";
    CoinType[CoinType["FILECOIN"] = 461] = "FILECOIN";
    CoinType[CoinType["FLOW"] = 539] = "FLOW";
    CoinType[CoinType["GAME"] = 101] = "GAME";
    CoinType[CoinType["GOCHAIN"] = 6060] = "GOCHAIN";
    CoinType[CoinType["GRS"] = 17] = "GRS";
    CoinType[CoinType["HCASH"] = 171] = "HCASH";
    CoinType[CoinType["HEDERA"] = 3030] = "HEDERA";
    CoinType[CoinType["HELIUM"] = 904] = "HELIUM";
    CoinType[CoinType["HPB"] = 269] = "HPB";
    CoinType[CoinType["HYCON"] = 1397] = "HYCON";
    CoinType[CoinType["ICON"] = 4801368] = "ICON";
    CoinType[CoinType["ICP"] = 223] = "ICP";
    CoinType[CoinType["IOTA"] = 4218] = "IOTA";
    CoinType[CoinType["IOV"] = 234] = "IOV";
    CoinType[CoinType["KIN"] = 2017] = "KIN";
    CoinType[CoinType["KOMODO"] = 141] = "KOMODO";
    CoinType[CoinType["KUSAMA"] = 434] = "KUSAMA";
    CoinType[CoinType["LBRY"] = 140] = "LBRY";
    CoinType[CoinType["LITECOIN"] = 2] = "LITECOIN";
    CoinType[CoinType["LISK"] = 134] = "LISK";
    CoinType[CoinType["MINA"] = 12586] = "MINA";
    CoinType[CoinType["MIX"] = 76] = "MIX";
    CoinType[CoinType["MONERO"] = 128] = "MONERO";
    CoinType[CoinType["MUSICOIN"] = 184] = "MUSICOIN";
    CoinType[CoinType["NANO"] = 165] = "NANO";
    CoinType[CoinType["NEAR"] = 397] = "NEAR";
    CoinType[CoinType["NEM"] = 43] = "NEM";
    CoinType[CoinType["NEO"] = 888] = "NEO";
    CoinType[CoinType["NERVOS"] = 309] = "NERVOS";
    CoinType[CoinType["NIMIQ"] = 242] = "NIMIQ";
    CoinType[CoinType["NIX"] = 400] = "NIX";
    CoinType[CoinType["NOS"] = 229] = "NOS";
    CoinType[CoinType["ONTOLOGY"] = 1024] = "ONTOLOGY";
    CoinType[CoinType["PARTICL"] = 44] = "PARTICL";
    CoinType[CoinType["PIRL"] = 164] = "PIRL";
    CoinType[CoinType["POA"] = 178] = "POA";
    CoinType[CoinType["POLKADOT"] = 354] = "POLKADOT";
    CoinType[CoinType["POSWALLET"] = 47] = "POSWALLET";
    CoinType[CoinType["QRL"] = 238] = "QRL";
    CoinType[CoinType["QTUM"] = 88] = "QTUM";
    CoinType[CoinType["RAVECOIN"] = 175] = "RAVECOIN";
    CoinType[CoinType["RIPPLE"] = 144] = "RIPPLE";
    CoinType[CoinType["RISE"] = 1120] = "RISE";
    CoinType[CoinType["REOSC"] = 2894] = "REOSC";
    CoinType[CoinType["RESISTANCE"] = 356] = "RESISTANCE";
    CoinType[CoinType["SOLANA"] = 501] = "SOLANA";
    CoinType[CoinType["STAKENET"] = 384] = "STAKENET";
    CoinType[CoinType["STACKS"] = 5757] = "STACKS";
    CoinType[CoinType["STRATIS"] = 105] = "STRATIS";
    CoinType[CoinType["STELLAR"] = 148] = "STELLAR";
    CoinType[CoinType["SHYFT"] = 7341] = "SHYFT";
    CoinType[CoinType["TEZOS"] = 1729] = "TEZOS";
    CoinType[CoinType["THUNDERCORE"] = 1001] = "THUNDERCORE";
    CoinType[CoinType["TOMO"] = 889] = "TOMO";
    CoinType[CoinType["TON"] = 607] = "TON";
    CoinType[CoinType["TRON"] = 195] = "TRON";
    CoinType[CoinType["UBIQ"] = 108] = "UBIQ";
    CoinType[CoinType["VECHAIN"] = 818] = "VECHAIN";
    CoinType[CoinType["WANCHAIN"] = 5718350] = "WANCHAIN";
    CoinType[CoinType["WAVES"] = 5741564] = "WAVES";
    CoinType[CoinType["ZCASH"] = 133] = "ZCASH";
    CoinType[CoinType["ZCLASSIC"] = 147] = "ZCLASSIC";
    CoinType[CoinType["ZCOIN"] = 136] = "ZCOIN";
    CoinType[CoinType["ZEN"] = 121] = "ZEN";
    CoinType[CoinType["ZILLIQA"] = 313] = "ZILLIQA";
    CoinType[CoinType["SUI"] = 784] = "SUI";
})(CoinType || (exports.CoinType = CoinType = {}));
